pkg/scheduler/api/resource_info.go

```diff
@@ -85,6 +85,32 @@ func NewResource(rl v1.ResourceList) *Resource {
 	return r
 }
 
+// ResFloat642Quantity transform resource quantity
+func ResFloat642Quantity(resName v1.ResourceName, quantity float64) resource.Quantity {
+	var resQuantity *resource.Quantity
+	switch resName {
+	case v1.ResourceCPU:
+		resQuantity = resource.NewMilliQuantity(int64(quantity), resource.DecimalSI)
+	default:
+		resQuantity = resource.NewQuantity(int64(quantity), resource.BinarySI)
```

Comment:
does the default works for memory? or will have to add a memory case for it.

Reply:
yes ,  the default works for memory by resource.NewQuantity